x-pack/plugins/fleet/server/services/epm/elasticsearch/transform/install.ts

```diff
@@ -703,9 +703,13 @@ async function handleTransformInstall({
   // start transform by default if not set in yml file
   // else, respect the setting
   if (startTransform === undefined || startTransform === true) {
-    await esClient.transform.startTransform(
-      { transform_id: transform.installationName },
-      { ignore: [409] }
+    await retryTransientEsErrors(
+      () =>
+        esClient.transform.startTransform(
+          { transform_id: transform.installationName },
+          { ignore: [409] }
+        ),
+      { logger, additionalResponseStatuses: [400] }
```

Comment:
By curiosity what is that 400 error that need to be retried?

Reply:
```
[ERROR][plugins.fleet] ResponseError: validation_exception
Caused by:
search_phase_execution_exception: all shards failed
Root causes:
validation_exception: Validation Failed: 1: Failed to test query, received status: SERVICE_UNAVAILABLE;
```

we also have https://github.com/elastic/security-team/issues/6165 to track fixing this more gracefully.